portal/common/lib/http/http_error_responses.ts

```diff
@@ -34,3 +36,17 @@ function Response404(message: String, secondaryMessage?: String): Response {
         },
     });
 }
+
+/**
+* Returns the html page that displays an alert to the user
+* regarding a mismatch between the aggregator response and
+* the blob hash (checksum).
+*/
+export function generateChecksumErrorResponse(): Response {
```

Comment:
Nit: `Checksum` as a term is weaker than `hash`. Should we just say that the hash does not match? (and rename the functions?)